packages/plugins/plugin-assistant/src/capabilities/toolkit.ts

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { AiTool, AiToolkit } from '@effect/ai';
+import { Effect, Schema } from 'effect';
+
+import { Capabilities, type PluginContext, contributes } from '@dxos/app-framework';
+import { Filter, Type } from '@dxos/echo';
+import { DatabaseService } from '@dxos/functions';
+import { ClientCapabilities } from '@dxos/plugin-client';
+import { SpaceCapabilities, getActiveSpace } from '@dxos/plugin-space';
+import { DataType } from '@dxos/schema';
+
+class SchemaToolkit extends AiToolkit.make(
+  AiTool.make('list-schemas', {
+    description: 'List the available schemas.',
```

Comment:
Retrieves the available schemas
(make from the point of view of the agent using the tool)
perhaps rename get-schemas?